examples/hello-triangle/main.rs

```diff
@@ -1,3 +1,4 @@
+use std::borrow::Cow;
```

Comment:
that's something I never did in my projects, but I wonder if it would make sense to go further and do `use std::borrow::Cow::Borrowed` ?